server/spec/auth/authenticate_user_spec.rb

```diff
@@ -0,0 +1,33 @@
+# spec/auth/authenticate_user_spec.rb
+require 'rails_helper'
+
+RSpec.describe AuthenticateUser do
+  # create test user
+  let(:user) { create(:api_user) }
+  # valid request subject
+  subject(:valid_auth_obj) { described_class.new(user.email, user.password) }
+  # invalid request subject
+  subject(:invalid_auth_obj) { described_class.new('foo', 'bar') }
```

Comment:
This name is also an abbreviation. Perhaps we can find a name which makes the code comment unnecessary?

Reply:
updated name